my-app/src/Pagrs/Home.tsx

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from "@tanstack/react-query";
+import { movieService } from "../Services/MovieService";
+import { Link } from "react-router-dom";
+import TruncateMarkup from "react-truncate-markup";
+
+const Home: React.FC<{}> = (props: any) => {
+  const { isError, isStale, isLoading, data, error } = useQuery(
+    ["movies"],
+    movieService.fetchAllMovies,
+    { staleTime: 3000 }
```

Comment:
go with a more default approach, use invalidateQuery when you want to fetch new data from backend